docs/pull_request_template.md

```diff
@@ -1,10 +1,10 @@
 [EVG-<number>](https://jira.mongodb.org/browse/EVG-<number>)
```

Comment:
Can we please update this as well. The spruce one auto linkifies the ticket
https://github.com/evergreen-ci/spruce/blob/main/.github/pull_request_template.md

Reply:
Same thing as the other comment, I don't really want to change the PR template meaningfully here.